src/ansys/geometry/core/tools/prepare_tools.py

```diff
@@ -168,6 +169,37 @@ def extract_volume_from_edge_loops(
             self._grpc_client.log.info("Failed to extract volume from edge loops...")
             return []
 
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def remove_rounds(self, faces: list["Face"]):
+        """Remove rounds from geometry.
+
+        Tries to remove rounds from geometry.  Faces to be removed are input to the method.
+
+        Parameters
+        ----------
+        round_faces : list[Face]
+            List of rounds faces to be removed
+
+        """
+        from ansys.geometry.core.designer.face import Face
+
+        if not faces:
+            self._grpc_client.log.info("No faces provided...")
+            return []
+
+        # Verify inputs
+        check_type_all_elements_in_iterable(faces, Face)
+
+        parent_design = get_design_from_face(faces[0])
+
+        response = self._prepare_stub.RemoveRounds(RemoveRoundsRequest(faces=faces))
+
+        if response.result:
+            parent_design._update_design_inplace()
+        else:
+            self._grpc_client.log.info("Failed to remove rounds...")
```

Comment:
Should we return the response status (i.e. boolean)? I think other tools are doing so.